src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -738,27 +750,19 @@ def __call__(
                     if callback is not None and i % callback_steps == 0:
                         callback(i, t, latents)
 
-        if output_type not in ["latent", "pt", "np", "pil"]:
-            deprecation_message = (
-                f"the output_type {output_type} is outdated. Please make sure to set it to one of these instead: "
-                "`pil`, `np`, `pt`, `latent`"
-            )
-            deprecate("Unsupported output_type", "1.0.0", deprecation_message, standard_warn=False)
-            output_type = "np"
-
-        if output_type == "latent":
+        if not output_type == "latent":
+            image = self.vae.decode(latents / self.vae.config.scaling_factor).sample
+            image, has_nsfw_concept = self.run_safety_checker(image, device, prompt_embeds.dtype)
+        else:
             image = latents
             has_nsfw_concept = None
```

Comment:
`has_nsfw_concept` is now either a list of bool or `None` (when didn't run safety_checker, i.e. `self.safety_checker is None` or `output_type=='latents'`)

I think this is consistent with the documentation as @pcuenca pointed out. And it is consistent with how it's used in current pipelines so I think it's ok

https://github.com/huggingface/diffusers/blob/7c1bb9ae81c97f6f10cf36784c17a51ebddeea08/src/diffusers/pipelines/stable_diffusion/__init__.py#L30-L32